src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -60,6 +60,7 @@
     private static final List<String> MULTI_TYPES =
             Arrays.asList("javax.management.openmbean.TabularData");
     private static final int MAX_RETURNED_METRICS = 350;
+    private static final int DEFAULT_INITIAL_REFRESH_BEANS_PERIOD = 150;
```

Comment:
IMO the `DEFAULT_INITIAL_REFRESH_BEANS_PERIOD` should be equal to `DEFAULT_REFRESH_BEANS_PERIOD `.